src/request/import/ImportWords.js

```diff
@@ -81,21 +83,20 @@ class ImportWords {
         });
 
         this._passwordSetter.on(PassphraseSetterBox.Events.ENTERED, () => {
-            const color = Iqons.getBackgroundColorIndex(
-                new Nimiq.Address(
-                    /*
-                     * this will either be unrelevant (UNKNOWN/PRIVATE_KEY) or correct (ENTROPY)
-                     * as `LoginFileIcon.setFileUnavailable(true)` was called in the former
-                     * and in the latter index 0 is the only index and contains the entropy
-                     */
-                    this._keys[0].addresses[0].address,
-                ).toUserFriendlyAddress(),
-            );
-            const colorString = LoginFile.CONFIG[color].name;
-            this._loginFileIcon.lock(`nq-${colorString}-bg`);
+            if (this._keys.entropy.addresses) {
+                const color = Iqons.getBackgroundColorIndex(
+                    new Nimiq.Address(
+                        // use color of first address as loginFile color
+                        this._keys.entropy.addresses[0].address,
+                    ).toUserFriendlyAddress(),
+                );
+                const colorString = LoginFile.CONFIG[color].name;
+                this._loginFileIcon.lock(`nq-${colorString}-bg`);
```

Comment:
This line should execute regardless of wether it is an entropy or a private key. In case of a private key the argument can be omitted, because it only closes the lock but does not change the color.

Reply:
Did I introduce a change here accidentally?

Reply:
In case of `this._keys.entropy === null` but `this._keys.privateKey !==null` the lock will not close.